app/javascript/onboarding/components/ProfileForm.jsx

```diff
@@ -0,0 +1,131 @@
+import { h, Component } from 'preact';
+import PropTypes from 'prop-types';
+
+import Navigation from './Navigation';
+import { getContentOfToken, updateOnboarding } from '../utilities';
+
+class ProfileForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleChange = this.handleChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+
+    this.state = {
+      summary: '',
+      location: '',
+      employment_title: '',
+      employer_name: '',
+      last_onboarding_page: 'personal info form', // i may need this i'm not sure exactly why yet
```

Comment:
Oh yeah, we should decide what we want to do about this...

I asked about this [on slack](https://dev-internal.slack.com/archives/C7GE84DEJ/p1586298508141500) yesterday as well.

Reply:
Oops I completely forgot that I left a comment there. Let me look at this now.

Reply:
I think that it doesn't hurt to leave it as is for now as there's potential for it to provide value.

Reply:
sg. how do you feel about prefacing it? so that it's easy to grep for? i was thinking that we could change new onboarding to be prefaced with `v2:`

Reply:
Updated!